src/routes/api.routes.js

```diff
@@ -10,6 +10,7 @@ router.get("/screenings", apiCtrl.getAllScreenings);
 router.get("/screenings/startpage", apiCtrl.startpageScreenings);
 router.get("/reviews", apiCtrl.reviews);
 router.get("/reviews/:id", apiCtrl.movieReviews);
+router.post("/reviews/:id", apiCtrl.review);
```

Comment:
In my opinion, this path is not great. According to common practice, the ID following `/reviews` will be thought by most to be the ID of a review, but you're actually using it as the ID of the movie that this review relates to.

I would have picked another path for this, e.g. `/movies/:id/reviews`.